### ISSUE TITLE
Not authorized error that only happens on my local machine

### ISSUE THREAD

**Issue by jasonperrone** [minimagick/minimagick#473]
I have a PDF file that I am trying to open with MiniMagick::Image.open( tmpfile ) where tmpfile is this PDF file in my /tmp directory.

On our production servers, that open has no problem, but on my local computer, I get this error? Doesn't anyone have any ideas?

`identify /tmp/mini_magick20181026-8840-17b4sst` failed with error:
identify-im6.q16: not authorized `/tmp/mini_magick20181026-8840-17b4sst' @ error/constitute.c/ReadImage/412.

**Comment by dverrier** [comment]
I had a similar problem. On my Ubuntu machine, I had to edit /etc/IMage-Magick-6/policy.xml to permit reading and writing pdfs. Apparently there was (or may be still is) an exploit of ghostscript and the policy was closed to prevent it. [https://imagemagick.org/script/security-policy.php](url) 
Discussion: [https://stackoverflow.com/questions/52861946/imagemagick-not-authorized-to-convert-pdf-to-an-image](url).
